objects/src/accounts/code.rs

```diff
@@ -180,22 +188,18 @@ impl Deserializable for AccountCode {
 // HELPER FUNCTIONS
 // ================================================================================================
 
-fn build_procedure_tree(procedures: &[Digest]) -> SimpleSmt<PROCEDURE_TREE_DEPTH> {
-    // order the procedure digests to achieve a reproducible tree
-    let procedures = {
-        let mut procedures = procedures.to_vec();
-        procedures.sort_by_key(|a| a.as_bytes());
-        procedures
-    };
-
-    SimpleSmt::<PROCEDURE_TREE_DEPTH>::with_leaves(
-        procedures
-            .iter()
-            .enumerate()
-            .map(|(idx, p)| (idx as u64, p.into()))
-            .collect::<Vec<_>>(),
-    )
-    .expect("failed to build procedure tree")
+fn procedures_as_elements(procedures: &[(Digest, Felt)]) -> Vec<Felt> {
```

Comment:
Here I add 2 helper functions enabling: 
- Retrieval of elements from procedures (Digest, Felt)
- Commit to these elements